var/spack/repos/builtin/packages/pigz/package.py

```diff
@@ -20,7 +20,7 @@ class Pigz(MakefilePackage):
     depends_on('zlib')
 
     def build(self, spec, prefix):
-        make()
+        make('CC=%s' % os.environ['CC'])
```

Comment:
```suggestion
        make('CC=cc')
```
or
```suggestion
        make('CC=%s' % self.compiler.cc)
```
Is a better way to do it